tools/Performance/DynamoPerformanceTests/DynamoPerformanceTests.csproj

```diff
@@ -40,122 +40,155 @@
     <AutoGenerateBindingRedirects>true</AutoGenerateBindingRedirects>
   </PropertyGroup>
   <ItemGroup>
-    <Reference Include="BenchmarkDotNet, Version=0.11.3.0, Culture=neutral, PublicKeyToken=aa0ca2f9092cefc4, processorArchitecture=MSIL">
-      <HintPath>..\..\..\src\packages\BenchmarkDotNet.0.11.3\lib\net46\BenchmarkDotNet.dll</HintPath>
+    <Reference Include="BenchmarkDotNet, Version=0.11.4.0, Culture=neutral, PublicKeyToken=aa0ca2f9092cefc4, processorArchitecture=MSIL">
+      <HintPath>packages\BenchmarkDotNet.0.11.4\lib\netstandard2.0\BenchmarkDotNet.dll</HintPath>
     </Reference>
     <Reference Include="CommandLine, Version=2.4.3.0, Culture=neutral, PublicKeyToken=de6f01bd326f8c32, processorArchitecture=MSIL">
-      <HintPath>..\..\..\src\packages\CommandLineParser.2.4.3\lib\netstandard2.0\CommandLine.dll</HintPath>
+      <HintPath>packages\CommandLineParser.2.4.3\lib\netstandard2.0\CommandLine.dll</HintPath>
+    </Reference>
+    <Reference Include="DynamoCoreTests">
+      <HintPath>..\..\..\bin\AnyCPU\Release\DynamoCoreTests.dll</HintPath>
```

Comment:
can this be parameterized for the config or we should only use release anyway?

Reply:
@aparajit-pratap what about using https://www.nuget.org/packages/DynamoVisualProgramming.Tests/

Reply:
maybe thats weird... it makes it harder to make changes to the system - devs will need to publish to a local nuget repo... curious what you think? Are the apis we need from this dll stable?

Reply:
I imagine that soon, we might need more APIs from the testing dll or core itself, for monitoring render time etc.

Reply:
It's preferable to use only Release but I'll parameterize it anyway.